### ISSUE TITLE
Move to mysqli

### ISSUE THREAD

**Issue by dionyziz** [dionyziz/endofcodes#89]

**Comment by vitsalis** [comment]
just change mysql_[connect,select] etc to mysqli_[connect,select] etc?

**Comment by dionyziz** [comment]
No, you have to use some OOP-style queries. Look at the docs. http://www.php.net/manual/en/book.mysqli.php
